### PR TITLE
Fixes fire_delay hard-cap being bypassable by spam clicking

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request
I forgot to also make this cap
## Why It's Good For The Game
Bugfix

## Changelog
:cl:
fix: Fixed fire-rate limits being bypassable by spam clicking(again)
/:cl:

### CODE REVIEW

**Comment by Humonitarian** [comment]
Weren't there a datum for it instead of the number 2?

**Comment by MLGTASTICa** [comment]
> Weren't there a datum for it instead of the number 2?

you mean , define , and yes tehre is , fixing that.

File: code/datums/datum_click_handlers.dm [hunk 1]
```diff
@@ -103,7 +103,7 @@
 	if(object)
 		target = object
 		shooting_loop()
-		time_since_last_init = world.time + reciever.burst_delay
+		time_since_last_init = world.time + reciever.fire_delay < GUN_MINIMUM_FIRETIME ? GUN_MINIMUM_FIRETIME : reciever.fire_delay
```
Comment (hyperioo) [inline code/datums/datum_click_handlers.dm:106]:
```suggestion
		time_since_last_init = world.time + (reciever.fire_delay < GUN_MINIMUM_FIRETIME ? GUN_MINIMUM_FIRETIME : reciever.fire_delay)
```

Comment (hyperioo) [reply code/datums/datum_click_handlers.dm:106]: Just in case it does something with the world.time +

**Pull request merged**